### PR TITLE
pangomm: migrate to Conan v2

### PR DESCRIPTION
**Pull request by valgur**

### CODE REVIEW

**Comment by RubenRBS** [comment]
Any updates on this one @valgur?

File: recipes/pangomm/all/conanfile.py [hunk 1]
```diff
@@ -1,101 +1,123 @@
...
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("pango/1.50.7")
+        self.requires("pango/1.50.14", transitive_headers=True)
```
Comment (RubenRBS) [inline recipes/pangomm/all/conanfile.py:62]: Where does this .14 come from? Latest available one in CCI is .10

File: recipes/pangomm/all/conanfile.py [hunk 2]
```diff
@@ -1,101 +1,123 @@
...
 
         # FIXME: temporary fix for dependency versions mismatch
         # once dependencies versions are bumped remove these requirements
-        self.requires("expat/2.4.8")
-        self.requires("zlib/1.2.12")
-        self.requires("glib/2.72.1")
+        self.requires("expat/2.5.0", override=True)
+        self.requires("zlib/1.2.13", override=True)
+        self.requires("glib/2.77.1", override=True)
```
Comment (RubenRBS) [inline recipes/pangomm/all/conanfile.py:68]: Can't use this in CCI, prefer to have PRs fixing the conflicts

File: recipes/pangomm/all/conanfile.py (hunk 1 above)
Comment (valgur) [reply recipes/pangomm/all/conanfile.py]: It's added by #18874.

**Comment by uilianries** [comment]
This PR depends on https://github.com/conan-io/conan-center-index/pull/18874 first.

**Comment by valgur** [comment]
Blocked by #22696.

**Pull request closed without merging**